### ISSUE TITLE
Why all labels are zeros?

### ISSUE THREAD

**Issue by NoshinXD** [HKUNLP/icl-ceil#8]
Hi! icl-ceil is very interesting. I'm trying to run run_ceil.sh, but facing some problems while running retriever_trainer.py. 

1. I am confused why all the labels are set to zeros when pair_wise==True inside collators.py. 
Are the p.label_ids in the function compute_metrics() and labels inside collators.py same? If I change the labels = torch.zeros(ctx_indices.shape[0]) to labels = torch.ones(ctx_indices.shape[0]), then np.unique(p.label_ids) changes from 0 to 1. But the shape of p.label_ids and labels are not same.

Looking forward to the response. Thank you!
